tests/test_io.py

```diff
@@ -393,19 +419,33 @@ def test_read_hdf_iterator(tmp_path: Path) -> None:
     ti.close()
 
 
-@pytest.mark.skipif(NUMPY20, reason="numpy 2.0 not compatible with Pytables")
 def test_hdf_context_manager(tmp_path: Path) -> None:
     path_str = str(tmp_path / str(uuid.uuid4()))
+    if sys.version_info < (3, 11) or sys.version_info >= (3, 14):
+        with pytest.raises(ImportError, match=MESSAGE_PYTABLE_MISSING):
+            _hdf_context_manager(path_str)
+    else:
+        _hdf_context_manager(path_str)
+
+
+def _hdf_context_manager(path_str: str) -> None:
     check(assert_type(DF.to_hdf(path_str, key="df", format="table"), None), type(None))
     with HDFStore(path_str, mode="r") as store:
         check(assert_type(store.is_open, bool), bool)
         check(assert_type(store.get("df"), DataFrame | Series), DataFrame)
 
 
-@pytest.mark.skipif(NUMPY20, reason="numpy 2.0 not compatible with Pytables")
```

Comment:
Wondering if we should just skip the test fro python314 for now instead of adding the version check (the same way we disabled the test for numpy above 2.0), what do you think? It will make things less verbose.